code_challenges/code_wars/find_the_odd_int.py

```diff
@@ -0,0 +1,11 @@
+# https://www.codewars.com/kata/54da5a58ea159efa38000836
+
+
+def odd_freq(numbers):
```

Comment:
You'll find that Code Wars often names their functions poorly.  A better name here would be `def least_freq_num(nums):`.

Moving forward think about if the name of the function is enough to describe what the intent of the function is.  You don't have to use the same function name here that you might have to in Code Wars due to their tests.